From 80d6b0c9477a3202e143f24a45aa63fad75e5de2 Mon Sep 17 00:00:00 2001
From: Chuck <fanck0605@qq.com>
Date: Sat, 11 Jul 2020 13:29:04 +0800
Subject: [PATCH] feeds: enable hell0world by default

Signed-off-by: Chuck <fanck0605@qq.com>
---
 feeds.conf.default | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/feeds.conf.default b/feeds.conf.default
index bae9813cfd..1be7d70a52 100644
--- a/feeds.conf.default
+++ b/feeds.conf.default
@@ -8,4 +8,4 @@ src-git freifunk https://github.com/freifunk/openwrt-packages.git
 #src-git management https://github.com/openwrt-management/packages.git
 #src-git oldpackages http://git.openwrt.org/packages.git
 #src-link custom /usr/src/openwrt/custom-feed
-#src-git helloworld https://github.com/fw876/helloworld
+src-git helloworld https://github.com/fw876/helloworld
-- 
2.17.1

